src/Network/Connection.php

```diff
@@ -232,6 +246,26 @@ public function setContext(array $context)
         $this->context = $context;
     }
 
+    /**
+     * Set the maximum number of bytes to write to a resource
```

Comment:
the comments of the 2 setter methods should describe why one would want that
e.g. that it seems to fix Amazon Queue